test/server-test.js

```diff
@@ -444,9 +461,11 @@ describe('SOAP Server', function () {
       assert.ifError(err);
       client.GetLastTradePrice({ tickerSymbol: 'trigger error' }, function (err, response, body) {
         assert.ok(err);
-        // what happens here (error and response ????)
-        // assert.strictEqual(err.response, response);
-        // assert.strictEqual(err.body, body);
+        assert.strictEqual(err.response, response);
+        assert.strictEqual(err.body, body);
+        // console.log(test.logMessages);
+        // assert.strictEqual(test.logMessages.length, 1);
+        // assert.notStrictEqual(test.logMessages[0].tyoe, {});
```

Comment:
I guess this and two lines above were used for debugging?  It would be good to remove unused code to keep it clean.